lib/stistools/inttag.py

```diff
@@ -1,5 +1,364 @@
-def inttag ():
+#! /usr/bin/env python
+import numpy as np
+from astropy.io import fits
+import astropy.stats
+from astropy import units as u
+from datetime import datetime as dt
 
-    #Integrate time tag event data into an image.
 
```

Comment:
Would be good to add more doc string stuff here to match the other stistools style.  Although, in tastis I've added the __doc__ title to the longer doc string so that it will show up in sphinx.

```
__doc__ = """
Analyze STIS target acquisition images. :func:`tastis` will print general
information about each input target acquisition image, and will analyze both
types of STIS target acquisitions: ACQs and ACQ/PEAKs

ACQ procedure is described in "STIS Onboard CCD Target Acquisitions" in the
STIS Instrument Handbook.  The ACQ/PEAK procedure is described in "Onboard
Target-Acquisition Peakups (ACQ/PEAK)" also in the STIS Instrument Handbook.

Target positions in global and local (subarray) coordinates and the total flux
of the target in the maximum checkbox during both acquistions phases (course
and fine) are displayed.

If update=True, keywords are added to the header to make problems easier to
locate in batch processing.  Warnings are given if the spt file is not present
when :func:`tastis` is run.

Examples
--------

:func:`tastis` with the default of update=False:

>>> import stistools
>>> stistools.tastis.tastis("ocmv0lw6q_raw.fits")

:func:`tastis` with update=True:

>>> import stistools
>>> stistools.tastis.tastis("ocmv0lw6q_raw.fits", update=True)
"""

__taskname__ = "tastis"
__version__ = "1.0"
__vdate__ = "14-June-2018"
__author__ = "Python: Sara Ogaz.  C code: R. Katsanis, R. Downes, " \
             "Phil Hodge. Original IDL code: George Hartig"
```

Reply:
Gotcha, sounds good. I'll add a __doc__ docstring.

Reply:
Added it. Let me know how this looks!

Reply:
Looks great, thanks @dougbrn!